### ISSUE TITLE
Breaking Change to `vmec2rpz` in `read_vmec.py`

### ISSUE THREAD

**Issue by connoramoreno** [aaroncbader/pystell_uw#7]
The updated implementation of `vmec2rpz` removes the toroidal angle `zeta` from the list of values returned from the method. All other class methods that use `vmec2rpz` rely on the toroidal angle being returned in their syntax, and thus errors result from their use.

To fix, one option is to revert `vmec2rpz` such that it returns the toroidal angle again (this seems simplest). Another option is to change the syntax of all other methods that rely on `vmec2rpz` such that they no longer require the toroidal angle to be returned.

Seeing as the name of the method implies that the toroidal angle is returned, I think it makes sense to implement the former.

**Comment by aaroncbader** [comment]
Yes, please put in a pull request to return z.  That change should not have been implemented

**Comment by connoramoreno** [comment]
I've got the change implemented locally, just trying to figure out GitHub permissions so that I can push my branch to this remote.
